extras/dfhack_extras.init

```diff
@@ -0,0 +1,21 @@
+## Extra tools, tweaks, and UI mods - additional to standard "dfhack.init".
+
+# keybindings for Lethosor's scripts
+keybinding add Alt-S@title settings-manager
+keybinding add Alt-S@dwarfmode/Default settings-manager
+keybinding add Alt-Q@jobmanagement manager-quantity
+keybinding add Alt-W@dwarfmode/QueryBuilding/Some/Workshop/Job gui/workflow
+keybinding add Alt-W@overallstatus "gui/workflow status"
+
+# enable Lethosor's scripts
+embark-tools enable sand sticky
```

Comment:
This one is in the default dfhack.init (except with `mouse` and without `sticky`)